logstash-core-event-java/lib/logstash/timestamp.rb

```diff
@@ -0,0 +1,28 @@
+# encoding: utf-8
```

Comment:
this feels strange that now Timestamp belongs in the core-event "gem". Almost begs for some other concept that captures these small set of logstash objects that can have multiple implementations.